test/audit/AuditServiceSpec.scala

```diff
@@ -51,6 +51,7 @@ class AuditServiceSpec extends AnyFreeSpec with ScalaCheckPropertyChecks with Be
     "must audit notification message event" in {
 
       val requestXml = <xml>test</xml>
+      val requestedXmlToJson = Json.parse("{\"channel\":\"api\",\"xml\":\"test\"}")
 
       val auditDetails = Json.toJson(AuditDetails(Json.obj("xml" -> "test")))
```

Comment:
Just to check, is this field still used? If not lets clean it up

Reply:
It is still used, its the response expected in the test, I could change it to `expectedResponse` if that makes it clearer?

Reply:
Was looking at the wrong bit :) Ill remove it!

Reply:
Annnddd removed! :)